yamllint/linter.py

```diff
@@ -223,9 +223,6 @@ def run(input, conf, filepath=None):
     :param input: buffer, string or stream to read from
     :param conf: yamllint configuration object
     """
-    if conf.is_file_ignored(filepath):
```

Comment:
Moved to `find_files_recursively` in `cli.py`.

Reply:
would this move break the ignore configuration when using yamllint as a Python library?

Reply:
@andrewimeson Mh, I wasn't aware that's a thing :shrug: 

WDYT of keeping this check then here and in CLI? :thinking: 

Would this work?